README.md

```diff
@@ -5,24 +5,46 @@
 ## Getting Started 🚀
 
 1. 🔑 Request an API Key (coming soon...)
-1. 🐦 Install Shorebird
+1. 🐦 Install the Shorebird CLI
 
    ```
-   dart pub global activate --source git https://github.com/shorebirdtech/shorebird --git-path packages/shorebird_cli
+   # Clone Shorebird
+   git clone https://github.com/shorebirdtech/shorebird
+
+   # Activate the Shorebird CLI
+   dart pub global activate --source path shorebird/packages/shorebird_cli
```

Comment:
This assumes you have the ~/.pub/bin in your path.  What's stopping us from having our own /bin in the install checkout?

Reply:
I just documented the current process but happy to adjust this and include our own `/bin`. I was holding off since we have a task to implement a proper installer.

Reply:
eseidel@erics-mbp counter % shorebird publish
Can't load Kernel binary: Invalid kernel binary format version.
Dart 2.19.4 is incompatible with your dependencies' SDK constraints. Please run "dart pub get" again.

Yet another problem with pub activate. 🤣   Such an unloved command.